keyboards/kbdfans/kbd6x/keymaps/vimwarrior_hhkb_tofu/readme.md

```diff
@@ -0,0 +1,9 @@
+# Vimwarrior HHKB Tofu Keymap
+
+Build Hex File:
+
+    make kbdfans/kbd6x:vimwarrior_hhkb_tofu
+
+Flash Keyboard
+
+    make kbdfans/kbd6x:vimwarrior_hhkb_tofu:dfu
```

Comment:
Actually, we've recently added a universal `:flash` target, and would prefer that it be used: 

```suggestion
    make kbdfans/kbd6x:vimwarrior_hhkb_tofu:flash
```

Reply:
Noted. I will replace it.

Reply:
WARNING: This board's bootloader is not specified or is not supported by the ":flash" target at this time.

Reply:
Many thanks will update it.